packages/ui/src/components/Table/utils.ts

```diff
@@ -26,3 +28,15 @@ export function isDetailsColumn(
 export function isRecord(obj: unknown): obj is Record<string, unknown> {
   return obj !== null && typeof obj === 'object'
 }
+
+export function getDefaultCellValue(row: TableRow, column: TableColumnApi, index: number) {
+  return column.formatter ? column.formatter(row, column, get(row, column.prop), index) : get(row, column.prop)
+}
+
+export function getCellTooltipContent(row: TableRow, column: TableColumnApi | TableActionColumnApi) {
+  if (!column.showOverflowTooltip || !isDefaultColumn(column)) {
+    return
```

Comment:
Why `return` and not `return ''`? In the second case the function always returns a string.